custom/icds_reports/ucr/data_sources/child_delivery_forms.json

```diff
@@ -229,6 +229,36 @@
         "is_nullable": true,
         "type": "expression",
         "column_id": "sex"
+      },
+        {
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "datatype": null,
+          "type": "property_path",
+          "property_path": ["case_open_child_health_3", "case", "@case_id"]
```

Comment:
out of curiosity why is it 3?

Reply:
Thats the "case tag"  from the case management settings https://india.commcarehq.org/a/icds-cas/apps/view/62a3e97f4b8e5d11977d44160865121b/form/48d1729300804739a193258b955f5bcf5ede4c0a/

I am not at all sure what it means or where those numbers come from, but it is used in creating the path to the case block